feat: add page title

Garfield-fr opened the pull request:
* Closes rero/rero-ils#3747.

To test use:
- https://github.com/rero/rero-ils-ui/pull/1346
- https://github.com/rero/ng-core/pull/739

Garfield-fr committed:
feat: add page title

* Closes rero/rero-ils#3747.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

PascalRepond commented:
# Tests

Public interface: the page title is displayed for a half-second and then is replaced by "-" in some views:
- Document search
- Patron profile viewer
- Patron profile editor
- "Change password" view
- Collections list
- Documents search results
- Entities search results

The title is OK in:
- Homepage
- Help pages
- Collection detailed view
- Document detailed view
- ILL request form
- Entity detailed view

Garfield-fr commented:
@PascalRepond  I've corrected the title on all the sections. I changed some titles to be in synch with the menus.

To test locally, you have to make Russian dolls.

> # Tests
> Public interface: the page title is displayed for a half-second and then is replaced by "-" in some views:
> 
> * Document search
> * Patron profile viewer
> * Patron profile editor
> * "Change password" view
> * Collections list
> * Documents search results
> * Entities search results
> 
> The title is OK in:
> 
> * Homepage
> * Help pages
> * Collection detailed view
> * Document detailed view
> * ILL request form
> * Entity detailed view
